src/code2vec.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import argparse
+from uuid import uuid4
+
+from extractors.paths import BagOfPathsExtractor
+from sourced.ml.transformers import UastDeserializer, Uast2BagFeatures, create_uast_source, \
+    UastRow2Document, Collector
+from sourced.ml.utils.engine import pipeline_graph, pause
+from algorithms.path_contexts import print_paths
+
+
+@pause
+def code2vec(args):
+    log = logging.getLogger("code2vec")
+    session_name = "code2vec-%s" % uuid4()
+    root, start_point = create_uast_source(args, session_name)
+
+    start_point \
+        .link(UastRow2Document()) \
+        .link(UastDeserializer()) \
+        .link(Uast2BagFeatures([BagOfPathsExtractor(args.max_length, args.max_width)])) \
+        .link(Collector()) \
+        .execute()
+
+    pipeline_graph(args, log, root)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    # sourced.engine args
+    parser.add_argument("-r", "--repositories",
```

Comment:
better to reuse arguments from `sourced.ml.cmd.args` like we do in `sourced-ml`.
https://github.com/src-d/ml/blob/master/sourced/ml/__main__.py#L110